src/applications/vaos/new-appointment/components/ReviewPage/CommunityCareSection.jsx

```diff
@@ -4,20 +4,24 @@ import ReasonForAppointmentSection from './ReasonForAppointmentSection';
 import PreferredDatesSection from './PreferredDatesSection';
 import PreferredProviderSection from './PreferredProviderSection';
 import SelectedProviderSection from './SelectedProviderSection';
+import { useSelector } from 'react-redux';
+import { selectFeatureCCIterations } from '../../../redux/selectors';
+
+export default function CommunityCareSection({ data, vaCityState }) {
+  const provider = data.communityCareProvider;
+  const hasProvider =
+    !!provider && !!Object.keys(data.communityCareProvider).length;
+  const myData = { ...data, hasCommunityCareProvider: hasProvider };
```

Comment:
Adding 'hasCoummunityCareProvider' flag to data object to be used with PreferredProviderSection. This flag is used to determine whether provider information is display or not.